RiotSwiftUI/Modules/Common/Avatar/Mock/MockAvatarService.swift

```diff
@@ -17,13 +17,14 @@
 import Foundation
 import Combine
 import DesignKit
+import UIKit
 
 @available(iOS 14.0, *)
 class MockAvatarService: AvatarServiceType {
     static let example: AvatarServiceType = MockAvatarService()
     func avatarImage(mxContentUri: String, avatarSize: AvatarSize) -> Future<UIImage, Error> {
         Future { promise in
-            promise(.success(Asset.Images.appSymbol.image))
+            promise(.success(Asset.Images.appSymbol.image))            
```

Comment:
We might be able to update 'SwiftGen' to return back SwiftUI `Image`s and we could remove the UIKit dependency and and use of UIImage here.

Will leave it for another PR.